internals/CHANGELOG.md

```diff
@@ -1,11 +1,27 @@
+# 0.4.3 - 2025-12-15
+
+- Add back in the `impl_from_infallible` macro
+
 # 0.4.2 - 2025-12-08
 
 - Move `impl_array_newtype` to internals [#5334](https://github.com/rust-bitcoin/rust-bitcoin/pull/5334)
 
 # 0.4.1 - 2024-10-18
 
+This was release violated semver rules, #3859 should have resulted in a `v0.5.0` release.
```

Comment:
In c18f8cc57f8e05a5e384e303d224aebeeaa97713:

`This was release` should be `This release`. And this is a run-on sentence.